addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParser.java

```diff
@@ -109,7 +136,7 @@ private void parseCategories(WappalyzerData wappalyzerData, String jsonStr) {
     }
 
     @SuppressWarnings("unchecked")
-    private void parseJson(WappalyzerData wappalyzerData, String jsonStr, boolean createIcons) {
+    private boolean parseJson(WappalyzerData wappalyzerData, String jsonStr, boolean createIcons) {
```

Comment:
I'd suggest keep void and just change the task to return null.

Reply:
`List<CompletableFuture<null>>` isn't accepted.

Reply:
May have got it.